pkgs/development/libraries/xdg-desktop-portal/default.nix

```diff
@@ -117,8 +123,12 @@ stdenv.mkDerivation (finalAttrs: {
     (lib.mesonEnable "systemd" enableSystemd)
   ] ++ lib.optionals (!enableGeoLocation) [
     "-Dgeoclue=disabled"
+  ] ++ lib.optionals (!finalAttrs.finalPackage.doCheck) [
+    "-Dpytest=disabled"
   ];
 
+  strictDeps = true;
```

Comment:
Needs `bash` in `buildInputs` for `/nix/store/yhvp35igng81yi7gd08h6q73nvm4yvrp-xdg-desktop-portal-1.18.4-installedTests/libexec/installed-tests/xdg-desktop-portal/test-document-fuse.sh` shebang

Reply:
`installedTests` need some other fixes too so i'll make a PR for those

Reply:
https://github.com/NixOS/nixpkgs/pull/344743